src/commands/utility/help.js

```diff
@@ -1,10 +1,21 @@
 const { SlashCommandBuilder } = require('discord.js');
 
 module.exports = {
-    data: new SlashCommandBuilder()
-        .setName('help')
-        .setDescription('Displays all commands.'),
-    async execute(interaction) {
-        await interaction.reply(`Still in development!`);
-    },
+	data: new SlashCommandBuilder()
+		.setName('help')
+		.setDescription('Get information about a command')
+		.addStringOption(option =>
+			option.setName('commands')
+				.setDescription('commands')
+				.setRequired(true)
+				.addChoices(
+					{ name: 'ban', value: 'bans user of choice' },
+					{ name: 'ping', value: 'replies pong' },
+					{ name: 'verify', value: 'get your roles' },
+				)),
+	async execute(interaction) {
+		const category = interaction.options.getString('commands');
+        	ephemeral: true;
```

Comment:
yes it was intentional, we should probs do that for most commands, espically verify. Creates a lot of unneeded backlog of messages

Reply:
Shouldn't really be that much backlog, commands won't be getting spammed.. and there's only 1 reply per command that just gets edited.

Would also be nice for moderators to be able to see what commands users are running. I imagine it might prompt other users to use the bot more often too.

IMO anyways, perhaps we can get a couple other opinions on this? @JamesOlwyn @unicornenjoyer